index.bs

```diff
@@ -524,15 +530,21 @@ Let <code>STI</code> be a corresponding [=shape tree instance=].
     <code class="vocab">st:ShapeTreeNonRDFResource</code>.
 * An { `ST` <code class="vocab">rdfs:label</code> `L` } [=arc=]
     indicates that there is exactly one corresponding [=managed resource=] 
-    `R` in `STI` and it has the name `L`.
+    `R` in `STI` and it has the name `L`. `L` is a <dfn>static resource</dfn>.
+
+Issue: Exactly one or at most one?
+Is a ShapeTree inconistent before a client gets around to creating static resources?
+Applies below as well.
+
```

Comment:
I think saying "at most" vs. "exactly" would keep cases where the static resource doesn't exist yet from implying inconsistency.